docs/_static/css/bannerStyles.css

```diff
@@ -39,7 +39,7 @@
 
 
 .topBanner {
-    background-color: rgb(31 208 130 / 70%); /* Change to your desired background color */
+    background-color: #2f3feabc; /* DTU BLUE */
```

Comment:
I'll merge this for now, but remember that we have a dtu-blue color, it would be better to use that reference directly (if possible).

Reply:
Yes! I expressed my self wrong in the comment. It is the DTU Blue but with a certain degree of transparency so that the 324 picture visible, so it is not _actually_ DTU Blue.